servers/tests/org.fusesource.ide.server.tests/src/main/java/org/fusesource/ide/server/tests/util/KarafMockRuntimeCreationUtil.java

```diff
@@ -36,6 +39,10 @@
 		KARAF_30 
 	};
 	
+	public static final String[] SUPPORTED_4X_RUNTIMES = new String[] {
```

Comment:
I am a bit baffled about this one. The constant is used in a different class which isn't a subclass of this one. So how could protected work here? Or do I miss something?

Reply:
see the description of the Rule: https://sonarcloud.io/organizations/default/rules#rule_key=squid%3AS2386

this is a vulnerability issue because it is a mutable object.

it is in test and we are not a program running on server so no problem to let it like that.

If we want to fix it, I think that we need to make it private and provide just a getter on an unmodifiable list